fit/core-reference/src/test/java/org/apache/syncope/fit/AbstractITCase.java

```diff
@@ -635,6 +668,38 @@ protected void updateLdapRemoteObject(
         }
     }
 
+    protected Pair<String, Set<Attribute>> prepareLdapAttributes(
```

Comment:
This method is used only by a single test, please move it back to the containing test class